Tests/Microsoft.AppCenter.Test.Functional/Analytics/TrackEventTest.cs

```diff
@@ -89,5 +90,116 @@ public async Task TrackEventWithPropertiesAsync()
             }
             Assert.Equal(1, httpNetworkAdapter.CallCount);
         }
+
+        [Fact]
+        public async Task TrackEventNormalFlowAsync()
+        {
+            var httpNetworkAdapter = new HttpNetworkAdapter(expectedLogType: "event");
+            DependencyConfiguration.HttpNetworkAdapter = httpNetworkAdapter;
+
+            // Start App Center.
+            AppCenter.UnsetInstance();
+            Analytics.UnsetInstance();
+            AppCenter.LogLevel = LogLevel.Verbose;
+            AppCenter.Start(_appSecret, typeof(Analytics));
+
+            // Pause Analytics module.
+            Analytics.Pause();
+
+            // Test TrackEvent.
+            Analytics.TrackEvent("TrackEvent 1");
+            Task.WaitAny(httpNetworkAdapter.HttpResponseTask, Task.Delay(5000));
```

Comment:
For readability, let's add a comment:
```
            // Wait for 5 seconds to allow batching happen (3 seconds), and verify nothing has been sent.
            Task.WaitAny(httpNetworkAdapter.HttpResponseTask, Task.Delay(5000));
            Assert.Equal(0, httpNetworkAdapter.CallCount);
```

Reply:
Same for https://github.com/microsoft/appcenter-sdk-dotnet/pull/1291/files#diff-b4a12c3dfee704bc3e6d1460129a6f42R168.